NodeDebug/Dockerfile.txt

```diff
@@ -1,5 +1,8 @@
 USER root
 
+# Debug
+ENV GRID_DEBUG true
```

Comment:
Debug in the images means that people can use VNC to see what is happening inside. Some people actually use this images for normal test runs. Please do not set this env var to true for the debug images (in the near future we will merge the images anyway).

Reply:
So the meaning is closer to `inspection` than `debug`. I'll correct it, thanks.